src/openvic-simulation/politics/Issue.cpp

```diff
@@ -73,19 +81,31 @@ bool IssueManager::add_reform(std::string_view identifier, ReformGroup const* gr
 		return false;
 	}
 
-	return reforms.add_item({ identifier, *group, ordinal });
+	if (!group->get_type().is_uncivilised() && technology_cost != 0) {
+		Logger::warning("Technology cost ", technology_cost, " found in civilised reform ", identifier, "!");
+	}
```

Comment:
```suggestion
	if (group->get_type().is_uncivilised()) {
		if (technology_cost <= 0) {
			Logger::warning("Non-positive technology cost ", technology_cost, " found in uncivilised reform ", identifier, "!");
		}
	} else if (technology_cost != 0) {
		Logger::warning("Non-zero technology cost ", technology_cost, " found in civilised reform ", identifier, "!");
	}
```
Sorry I should've been clearer, I meant change the message, we want a warning if there's a non-zero tech cost for a civilised one OR if there's a zero tech cost for an uncivilised one (since 0 tech cost will likely mean the tech cost key was missing from the reform).